pyproject.toml

```diff
@@ -15,6 +15,7 @@ astor = "^0.8.1"
 openai = "^0.27.5"
 ipython = "^8.13.1"
 matplotlib = "^3.7.1"
+click = "^8.1.3"
```

Comment:
Isn't it a dev dependency?

Reply:
I'm still fairly new to contributing to open source projects, I'm unsure if its a dev dependency; should i move it from dependencies to dev dependencies?